vtkext/private/module/vtkF3DGenericImporter.cxx

```diff
@@ -213,6 +212,16 @@ void vtkF3DGenericImporter::ImportActors(vtkRenderer* ren)
     // Add filter outputs to mapper inputs
     pipe.PolyDataMapper->SetInputConnection(pipe.PostPro->GetOutputPort(0));
     pipe.PointGaussianMapper->SetInputConnection(pipe.PostPro->GetOutputPort(1));
+
+    for (auto& reader : this->Pimpl->Readers)
+    {
+      if (!reader.VolumeMapper)
+      {
+        reader.VolumeMapper = vtkSmartPointer<vtkSmartVolumeMapper>::New();
+        reader.VolumeMapper->SetRequestedRenderModeToGPU();
+      }
+    }
```

Comment:
you can remove the loop and just do something like:

```
       pipe.VolumeMapper = vtkSmartPointer<vtkSmartVolumeMapper>::New();
       pipe.VolumeMapper->SetRequestedRenderModeToGPU();
```

Reply:
thanks for the review @mwestphal I have updated the files to reflect the changesssss